### PR TITLE
Create loremipsum.txt

### PR DESCRIPTION
**Pull request by notthetup**
This pull request is to help learn about reviewing and commenting on pull requests.

Please review and comment on this pull request.

Context : https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-proposed-changes-in-a-pull-request

❤ Thank you!

### CODE REVIEW

**Comment by AmyGibSubnero** [comment]
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque consequat dapibus malesuada. Maecenas hendrerit pulvinar tellus suscipit iaculis. Pellentesque eget molestie ligula. Nullam facilisis nisl ac neque mollis consectetur. Etiam commodo augue orci, non vulputate lectus porttitor at. Suspendisse tincidunt finibus est in lobortis. Vivamus id nisl et purus finibus fringilla. Ut non ipsum porttitor, bibendum metus at, dignissim massa. Fusce scelerisque facilisis tristique. Morbi justo leo, fermentum ac ante non, dignissim mattis quam. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Curabitur fermentum dolor sit amet elit sollicitudin accumsan. Etiam quis libero justo.

File: loremipsum.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
...
+
+Duis sem velit, tincidunt nec ipsum vitae, iaculis pharetra risus. Fusce a felis ac ante cursus bibendum. Donec sit amet turpis at ipsum porta dapibus et ac libero. Aliquam ac leo quis eros dapibus feugiat eget suscipit lorem. Ut scelerisque libero a mi tincidunt, sed tempor magna tristique. Proin non enim non ex vehicula feugiat in pretium nunc. Proin vulputate dolor at elit blandit, nec porttitor ipsum fringilla. In tincidunt libero non dui malesuada, at ultrices tortor accumsan. Nam non purus hendrerit, luctus erat non, efficitur est. Integer convallis ipsum in felis varius, sit amet luctus quam volutpat. Suspendisse augue sapien, hendrerit sed lacus a, tempor aliquet orci.
```
Comment (AmyGibSubnero) [inline loremipsum.txt]: Nullam vehicula justo orci, vitae sollicitudin risus ullamcorper sit amet.

Comment (notthetup) [reply loremipsum.txt]: Integer convallis ipsum in felis varius, sit amet luctus quam volutpat.

**Review by Florie25 — COMMENTED** [review]
Do you then need to modify the original document as per comments?

**Comment by notthetup** [comment]
I can do that. Let me do that.

Now you see it' says "Updated".

**Pull request closed without merging**